server.go

```diff
@@ -19,6 +20,32 @@ func fallback(w http.ResponseWriter, r *http.Request, reason string) {
 	http.Redirect(w, r, location, 302)
 }
 
+func getRedirect(txt []string, url string) (*Redirect, error) {
+	catch_alls := make(map[string]*Config)
```

Comment:
Also, since `map` doesn't guarantee order, can we use a list/slice with `append` instead?